### DISCUSSION TITLE
Secure Supply Chain

### DISCUSSION THREAD

**Discussion by agracey** [epinio/epinio#1861] [General]
There is increasing focus on the security of container supply chains. As this is a reasonably hard problem to solve for most people, building it into the PaaS would make it easier. 

https://docs.google.com/document/d/1VURD9rdEhiuqPdixhEozkHw01Tk6e2AaJVjBK3pK6Zc

**Comment by gitricko** [comment]
Would love to know more how Epinio address some of these requirements
